Python/Product/Django/TemplateParsing/DjangoFormatDefinitions.cs

```diff
@@ -24,8 +24,8 @@
 namespace Microsoft.PythonTools.Django.TemplateParsing {
     [Export(typeof(EditorFormatDefinition))]
     [ClassificationType(ClassificationTypeNames = DjangoPredefinedClassificationTypeNames.TemplateTag)]
-    [Name(DjangoPredefinedClassificationTypeNames.TemplateTag)]
-    [DisplayName(DjangoPredefinedClassificationTypeNames.TemplateTag)]
+    [Name(DjangoPredefinedClassificationTypeNames.TemplateTag)] // TODO: Localization - this string appears in fonts page in tools options
+    [DisplayName(DjangoPredefinedClassificationTypeNames.TemplateTag)] // TODO: Localization - not sure if this is used, does not affect fonts page
```

Comment:
On French VS, names in the font page are translated. Will have to investigate how it's done later (I'll review to all TODOs).